src/lib/platform/PortalInputCapture.h

```diff
@@ -58,9 +58,9 @@ class PortalInputCapture {
     {
         reinterpret_cast<PortalInputCapture*>(data)->cb_session_closed(session);
     }
-    static void cb_disabled_cb(XdpInputCaptureSession *session, gpointer data)
+    static void cb_disabled_cb(XdpInputCaptureSession *session, GVariant* options, gpointer data)
     {
-        reinterpret_cast<PortalInputCapture*>(data)->cb_disabled(session);
+        reinterpret_cast<PortalInputCapture*>(data)->cb_disabled(session, options);
```

Comment:
semi-unrelated to this patch but it looks like the `cb_activated_cb` and `deactivated` needs fixing too? From the [docs](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.InputCapture.html#org-freedesktop-portal-inputcapture-activated) the `activation_id` is inside the `options` array so on a quick glance the current code is similarly buggy.

Reply:
Nope, i check the signature of activated and deactivated is good because libportal gently extract for us `activation_id` from `option`.
[libportal/inputcapture.c#L420](https://github.com/flatpak/libportal/blob/08c86829ad92d417ffd760f91af3ba9e2f7c52b8/libportal/inputcapture.c#L420)